pygmt/enums.py

```diff
@@ -0,0 +1,39 @@
+"""
+Enumerations for PyGMT.
+"""
+
+from enum import IntEnum
+
+
+class GridID(IntEnum):
+    """
+    Enum for the GMT grid format ID.
+
+    These enums are defined in 'gmt_grdio.h'.
+    """
+
+    GMT_GRD_UNKNOWN_FMT = 0  # If grid format cannot be auto-detected
+    GMT_GRID_IS_BF = 1  # GMT native, C-binary format (32-bit float)
```

Comment:
Since these enumerations are accessed like `GridID.GMT_GRID_IS_BF`, maybe we should define shorter enumerations like `GridID.BF` instead.

Reply:
Done in ce3c55cd71c65919c94861790be3f3ecb1c9e98e.